library/nats

```diff
@@ -28,23 +28,23 @@ Architectures: windows-amd64
 Directory: 2.10.x/nanoserver-1809
```

Comment:
This `2.10*nanoserver-1809` entry should also be getting the `2.10.3` bump in its tags.

Reply:
Thanks @yosifkit, just updated.